lib/analyzers/jspm/index.js

```diff
@@ -55,15 +55,15 @@ function getJspmCreditsPerCategory( projectPath, category ) {
 *
 * @return {Array}              list of credits
 */
-function getJspmCredits( projectPath ) {
+function getCredits( projectPath ) {
   var credits    = [];
   var categories = [ 'npm', 'github/components' ];
```

Comment:
I didn't catch this earlier... only checks the `github/components` path.  It should search all paths directories under `github`.